component/class/defaults.yml

```
@@ -0,0 +1,12 @@
+parameters:
+  cloudscale_metrics_collector:
+    namespace: 'appuio-cloud-reporting'
+    secrets:
+      cloudscale:
+        stringData:
+          token: 'replace-this-by-the-real-cloudscale-token'
```

I would not provide a random placeholder, but an actual vault reference, so that users can put the token into Vault at the default location.
```suggestion
    secrets:
      cloudscale:
        stringData:
          token: "?{vaultkv:${cluster:tenant}/${cluster:name}/cloudscale-metrics-collector/token}"
```

---

Fixed in https://github.com/vshn/cloudscale-metrics-collector/pull/10. I also changed the example in the documentation.